### PR TITLE
Dockerize prerelease website

### PR DESCRIPTION
**Pull request by ramereth**
This will allow the OSL to manage this as a docker container and handle updates
better in the long term. Some key changes:

- Add weekly dependabot checks
- Add requirements.txt
- Add basic CI
- Build and release container images on ghcr.io which will be used in the
  cookbook
- Do a weekly build regardless of any dependabot bumps
- Deploy website using gunicorn instead of mod_python
- Create wsgi.py for gunicorn

Signed-off-by: Lance Albertson <lance@osuosl.org>

### CODE REVIEW

**Comment by nuclearsandwich** [comment]
I bootstrapped GitHub Actions on the default branch but that hasn't given me the Actions popup in this PR to allow new contributors. I've relaxed that setting for now so hopefully a subsequent push will be enough to kick that off without an additional review. If it doesn't, we might have to split the PR into adding the workflow knowing it will fail, and then fixing up the workflow.

**Comment by ramereth** [comment]
@nuclearsandwich for some reason I can't get the ci.yml job to run. Any ideas why that might be?

**Comment by nuclearsandwich** [comment]
> @nuclearsandwich for some reason I can't get the ci.yml job to run. Any ideas why that might be?

Actions was added to GitHub after I left the company so I have no leftover insights.
My only guess is that Actions were not enabled on the default branch when the PR was

**Comment by nuclearsandwich** [comment]
Closing and re-opening the PR appears to have kicked that which needed kicking.

**Comment by ramereth** [comment]
@nuclearsandwich oh well, if this looks good, please go ahead and merge so that I can test the deployment side of it.

File: .github/workflows/container.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    - cron: '45 18 * * 3'
```
Comment (nuclearsandwich) [inline .github/workflows/container.yml:10]: 6:45PM on Wednesday.

File: entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+exec gunicorn \
+  -u nobody \
+  -g nobody \
+  --access-logfile '-' \
+  --error-logfile '-' \
+  --log-file "-" \
+  --access-logformat '%({x-forwarded-for}i)s %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"' \
+  --forwarded-allow-ips="140.211.9.104,2605:bc80:3010:104::8cd3:968" \
```
Comment (nuclearsandwich) [inline entrypoint.sh:9]: I may send a follow-up PR that breaks these values out into a variable so they can be described in a comment for future me who forgets what these IPs are.

**Review by nuclearsandwich — APPROVED** [review]

**Pull request merged**